docs/app/partials/layout-container.tmpl.html

```diff
@@ -50,6 +35,111 @@
   </docs-demo>
 
   <p>
-    See the <a href="layout/options">layout options page</a> for information on responsive layouts and other options.
+      Note that <code>layout</code> only affects the flow direction for that container's <b>immediate</b> children.
+    </p>
+
+  <hr>
+
+  <br/>
+  <h3>Layouts and Responsive Breakpoints</h3>
+
+  <p>
+    As discussed in <a href="layout/introduction">Layout Introduction page</a> you can
```

Comment:
Suggestion: Add `the` - "As discussed in **the** `<a href=...`"

or just remove the word "page" to make it a bit more sentence-like.